### PR TITLE
Add missing sections to release notes

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Add missing sections on Tiller to the latest release notes

### CODE REVIEW

File: content/blog-posts/2020-01-09-release-notes-1.9/index.md [hunk 1]
```diff
@@ -86,6 +86,10 @@ In this release, we introduced Knative Eventing Mesh as an alpha feature which y
 
 After introducing Gardener cluster provisioning through the Kyma CLI in the previous release, in this release, we added the official documentation for this feature. The instructions for provisioning a Gardener cluster on GCP or Azure are now based on Kyma CLI. Check it out in [this](https://kyma-project.io/docs/1.9/root/kyma/#installation-install-kyma-on-a-cluster-prepare-the-cluster) document under the **Gardener** tab.
 
+### Tiller now keeps a finite number of revisions
+
+Up until now, Tiller kept an infinite number of revisions. This resulted at times in Helm not being able to upgrade charts after many retries. Setting a limit to the number of revisions, resolved these issues. Another advantages of keeping only a few revisions is lower memory consumption for controllers listening on ConfigMap resources and simply fewer resources for Kubernetes to manage.
```
Comment (bszwarc) [inline content/blog-posts/2020-01-09-release-notes-1.9/index.md]:
```suggestion
Up until now, Tiller kept an infinite number of revisions. This resulted at times in Helm not being able to upgrade charts after many retries. Setting a limit to the number of revisions, resolved these issues. Another advantage of keeping only a few revisions is lower memory consumption for controllers listening on ConfigMap resources and fewer resources for Kubernetes to manage.
```

File: content/blog-posts/2020-01-09-release-notes-1.9/index.md [hunk 2]
```diff
@@ -160,16 +164,26 @@ A temporary workaround for the issue is as follows:
...
 
 >**NOTE:** Old CustomResourceDefinitions (CRDs) are not removed automatically during the upgrade process. Leaving the CRDs does not influence the upgrade. These CRDs will be removed with 1.10 release. However, you can remove them manually anytime using this command:
 >
 >```bash
 >kubectl delete crd clusterdocstopics.cms.kyma-project.io && kubectl delete crd docstopics.cms.kyma-project.io && kubectl delete crd clusterbuckets.assetstore.kyma-project.io && kubectl delete crd buckets.assetstore.kyma-project.io && kubectl delete crd clusterassets.assetstore.kyma-project.io && kubectl delete crd assets.assetstore.kyma-project.io
 >```
+
+### Tiller
+
+As we modified the Tiller Deployment by adding a new environment variable that sets the Tiller revisions limit, you should apply the new Tiller resource before performing the upgrade:
```
Comment (bszwarc) [inline content/blog-posts/2020-01-09-release-notes-1.9/index.md]:
```suggestion
As we modified the Tiller Deployment by adding a new environment variable that sets the Tiller revisions limit, apply the new Tiller resource before performing the upgrade:
```

File: content/blog-posts/2020-01-09-release-notes-1.9/index.md [hunk 3]
```diff
@@ -21,12 +21,12 @@ See the overview of all changes in this release:
 - [Compass](#compass) - Provisioner enhanced, Application Templates added, redesigned list view in the Compass UI, and more
 - [Console](#console) - Improved Addons Configuration view
 - [Eventing](#eventing) - Knative Eventing Mesh available for testing purposes
-- [Installation & Documentation](#installation-documentation) - Gardener provisioning based on Kyma CLI documented
+- [Installation & Documentation](#installation-documentation) - Gardener provisioning based on Kyma CLI documented, a finite number of revisions set in Tiller
```
Comment (bszwarc) [inline content/blog-posts/2020-01-09-release-notes-1.9/index.md]:
```suggestion
- [Installation & Documentation](#installation-documentation) - Gardener provisioning based on Kyma CLI documented, a finite number of revisions set in Tiller
```
do we use articles here? 🤔

Comment (kazydek) [reply content/blog-posts/2020-01-09-release-notes-1.9/index.md]: true, I will remove it

File: content/blog-posts/2020-01-09-release-notes-1.9/index.md [hunk 4]
```diff
@@ -160,16 +164,26 @@ A temporary workaround for the issue is as follows:
...
 >```bash
 >kubectl delete crd clusterdocstopics.cms.kyma-project.io && kubectl delete crd docstopics.cms.kyma-project.io && kubectl delete crd clusterbuckets.assetstore.kyma-project.io && kubectl delete crd buckets.assetstore.kyma-project.io && kubectl delete crd clusterassets.assetstore.kyma-project.io && kubectl delete crd assets.assetstore.kyma-project.io
 >```
+
+### Tiller
+
+As we modified the Tiller Deployment by adding a new environment variable that sets the [Tiller revisions limit](#finite-number-of-revisions-in-tiller), apply the new Tiller resource before performing the upgrade:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/$KYMA_VERSION/installation/resources/tiller.yaml
```
Comment (kubadz) [inline content/blog-posts/2020-01-09-release-notes-1.9/index.md]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/1.9.0/installation/resources/tiller.yaml
```

**Pull request merged**